packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -405,7 +405,10 @@ abstract class Widget extends DiagnosticableTree {
   /// A short, textual description of this widget.
   @override
   String toStringShort() {
-    return key == null ? '$runtimeType' : '$runtimeType-$key';
+    if (!kReleaseMode) {
```

Comment:
Open for suggestions here - we could skip this for now, but it makes sense to deal with this in widget since users will likely never override toStringShort.